oresat_configs/base/imu.yaml

```diff
@@ -201,6 +201,87 @@ objects:
         low_limit: -8192
         high_limit: 8192
 
+  - index: 0x4007
+    name: magnetorquer_current_x
+    object_type: record
+    description: Magnetorquer Current Feedback in uA
+    subindexes:
+      - subindex: 0x1
+        name: current_feedback
+        data_type: int32
+        description: current feedback
+        access_type: ro
+
+      - subindex: 0x2
+        name: current_set
+        data_type: int32
+        description: Set current
+        access_type: rw
+
+  - index: 0x4008
+    name: magnetorquer_current_y
+    object_type: record
+    description: Magnetorquer Current Feedback in uA
+    subindexes:
+      - subindex: 0x1
+        name: current_feedback
+        data_type: int32
+        description: current feedback
+        access_type: ro
+
+      - subindex: 0x2
+        name: current_set
+        data_type: int32
+        description: Set current
+        access_type: rw
+
+  - index: 0x4009
+    name: magnetorquer_current_z
+    object_type: record
+    description: Magnetorquer Current Feedback in uA
+    subindexes:
+      - subindex: 0x1
+        name: current_feedback
+        data_type: int32
+        description: current feedback
+        access_type: ro
+
+      - subindex: 0x2
+        name: current_set
+        data_type: int32
+        description: Set current
+        access_type: rw
+
+  - index: 0x4010
+    name: magnetorquer_pwm_percent
+    object_type: record
+    description: Magnetorquer PWM Percent
+    subindexes:
+      - subindex: 0x1
+        name: x
+        data_type: int16
+        description: X axis
+        access_type: ro
+        low_limit: 0
+        high_limit: 10000
+
+      - subindex: 0x2
+        name: y
+        data_type: int16
+        description: Y axis
+        access_type: ro
+        low_limit: 0
+        high_limit: 10000
+
+      - subindex: 0x3
+        name: z
+        data_type: int16
+        description: Z axis
+        access_type: ro
+        low_limit: 0
+        high_limit: 10000
```

Comment:
Can we change this one  magnetorquer record with a bunch of subindexes

Something like:
````
magnetorquer
   current_x
   current_y
   current_z
   current_x_setpoint
   ...
   pwm_z
````